internal/utils/csi.go

```diff
@@ -119,6 +121,27 @@ var EtcSelinuxVolume = corev1.Volume{
 	},
 }
 
+func LogDirVolume(LogHostPath, CsiComponentName string) corev1.Volume {
+	return corev1.Volume{
+		Name: csiLogDirectoryVolumeName,
+		VolumeSource: corev1.VolumeSource{
+			HostPath: &corev1.HostPathVolumeSource{
+				Path: fmt.Sprintf("%s/log/%s", LogHostPath, CsiComponentName),
+				Type: ptr.To(corev1.HostPathDirectoryOrCreate),
+			},
+		},
+	}
+}
+
+func LogSidecarDirVolume(LogHostPath, CsiComponentName string) corev1.Volume {
```

Comment:
The arguments for this volume function are not used. 
Why is this a function and not a static var volume?